xml/metering-metering_notifications.xml

```diff
@@ -89,7 +91,7 @@ sinks:
    <emphasis role="bold">Sample Ceilometer Notification Agent(notification
    agent) file:</emphasis>
   </para>
-<screen>#File:    ~/opt/stack/service/ceilometer-common/etc/pipeline.yaml
+<screen>#File:    ~/opt/stack/service/ceilometer-agent-notification/etc/pipeline-agent-notification.yaml
 ---
```

Comment:
I see you removed the ~ above but not here.  Is there a purpose for the ~ in this documentation format?

Reply:
Thanks! Added ~ back in (not sure if it has something to do with formatting)